packages/core/src/components/ozon-content/nodes/figuur.node.tsx

```diff
@@ -39,12 +39,31 @@ export class OzonContentFiguurNode implements OzonContentNode {
     imageElement.width = naturalWidth * (schaal / 100);
   }
 
-  onImageLoad(event: Event, schaal?: number) {
-    if (event.target instanceof HTMLImageElement && schaal) {
-      this.setImageDimensions(event.target, schaal);
+  onImageLoad(event: Event, breedte?: number, hoogte?: number, schaal?: number) {
+    if (event.target instanceof HTMLImageElement) {
+      if (schaal) {
+        this.setImageDimensions(event.target, schaal);
+      }
+      if (!schaal && breedte && hoogte) {
+        // Adjust aspect-ratio by naturalWidth and naturalHeight once it is loaded
+        const { naturalHeight, naturalWidth } = event.target as HTMLImageElement;
+        event.target.style.aspectRatio = (naturalWidth / naturalHeight).toString();
+      }
```

Comment:
Dit zorgt er dus voor dat uiteindelijk altijd de `aspect-ratio` gebaseerd is op de juiste breedte en hoogte van de afbeelding. Dus in het zeldzame geval een afbeelding een onjuiste breedte en/of hoogte heeft in de STOP xml, dan zal de afbeelding een andere afmeting aannemen als de afbeelding is geladen.